templates/emails/group.monthlyreport.hbs

```diff
@@ -81,14 +81,17 @@ We hope it's useful. Don't hesitate to send us your feedback by replying to this
 {{/each}}
 </table>
 
-<h2>Newest sponsors on OpenCollective</h2>
+{{#if topBackers}}
+<h2>Latest related backers</h2>
```

Comment:
Hard to parse. How about "Recent sponsors on Open Collective"? We don't need to tell them that we filtered to related collectives.